content/module-reference/utility-modules/darkroom/high-quality-processing.md

```diff
@@ -0,0 +1,29 @@
+---
+title: high quality processing
+id: hq-processing
+applicable-version: 4.8
+tags:
+view: darkroom
+---
+
+Click the ![high quality processing icon](./high-quality-processing/high-quality-processing-icon.png#icon) icon to activate high quality processing in the pixelpipe.
+
+Darktable pixelpipes process the modules in slightly different ways, the thumbnail pipe as an example is optimized for speed.
+
+The darkroom pixelpipes for the main canvas and the second monitor try to keep resources within limits by downscaling and by
+restricting the processed data to what is actually displayed - in darktable speak this is the "region of interest" (ROI).
+
+Yet - for exports we have to process the whole image - likely without downscaling - resulting in much larger processing times
```

Comment:
Use a double-hyphen `--` in markdown for dashes, though I would replace the first one with a comma